src/sentry/auth/providers/github/provider.py

```diff
@@ -58,7 +56,7 @@ def build_identity(self, state):
         }
 
     def refresh_identity(self, auth_identity):
-        client = GitHubClient(self.client_id, self.client_secret)
+        client = GitHubClient()
         access_token = auth_identity.data["access_token"]
```

Comment:
@slohmes To me, it would be much more logical to make `GitHubClient` take the `access_token` as an argument since we need it for every action. Granted, it's outside the scope of the PR so I'll leave that as an optional suggestion.

Reply:
That makes sense to me. Here's what that'd look like: https://github.com/getsentry/sentry/pull/19146 -- lmk what you think.

Reply:
👍 to this change.